packages/experimental/Avatar/src/JSAvatar.styling.ts

```diff
@@ -11,7 +11,7 @@ const nameMap: { [key: string]: string } = {
   end: 'flex-end',
 };
 
-export const avatarStates: (keyof JSAvatarTokens)[] = ['circular', 'square'];
+export const avatarStates: (keyof JSAvatarTokens)[] = ['circular', 'square', 'inactive'];
```

Comment:
why 'inactive' and not 'active'?

Reply:
I think because active is the state where we don't need to apply any special effects, while we need an opacity effect for inactive.

Reply:
Yes, we have opacity only for inactive. Active and unset have opacity 1.